mlbstatsapi/models/game/livedata/plays/play/playrunner/attributes.py

```diff
@@ -0,0 +1,117 @@
+from typing import Union, Optional
+from dataclasses import dataclass
+
+from mlbstatsapi.models.people import Person
+
+@dataclass
+class RunnerCreditsPosition:
```

Comment:
Have you seen this anywhere else? This has the same attributes as PrimaryPosition. I feel we should just rename it to Position and use the class here and for Person, Player, or where ever else.

Reply:
We don't have to address this now, but we should take note.

Reply:
Feel free to resolve this your self

Reply:
Good catch Matt! Changed PrimaryPosition in people.attributes to Position and got rid of RunnerCreditsPosition.